examples/Handpose-p5-image/sketch.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2023 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+let handpose;
+let video;
+let hands = [];
+
+function preload() {
+  //load the image to be detected
+  img = loadImage("hand.jpg");
+  // Load the handpose model.
+  handpose = ml5.handpose();
+}
+
+function setup() {
+  createCanvas(640, 480);
+  fill(0, 255, 0);
+  noStroke();
+  // Draw the image
+  image(img, 0, 0);
+  // start detecting hands from the webcam video
+  handpose.detect(img, gotHands);
+}
+
+// Callback function for when handpose outputs data
+function gotHands(results) {
+  // save the output to the hands variable
+  hands = results;
```

Comment:
For this example, if the drawing is happening in `gotHands()` I don't think we need a global variable. I would either just draw from `results` only or move drawing to the `draw()` loop? Not sure which is a better foundational example? You could also name the argument to `gotHands()` as `hands` if you like?

Reply:
Thank you for this suggestion! I would lean toward moving the drawing to the `draw()` loop since the students might want to modify the example into something more interesting and it would be easier if the drawing is already in `draw()`.